docs/README.android

```diff
@@ -201,19 +177,23 @@ on with the Android toolchain and creating an Android Application Package
 5.1. Building dependencies
 --------------------------------------------------------------------
 
-    $ cd $HOME/kodi-android/tools/depends
+    $ cd $HOME/kodi-krypton/tools/depends
     $ ./bootstrap
-    $ ./configure --help
+    $ ./configure --with-tarballs=/$HOME/xbmc-tarballs \
+    --host=arm-linux-androideabi --with-sdk-path=/$HOME/android-sdk-linux \
+    --with-ndk=/$HOME/android-ndk-r12b \
+    --with-toolchain=/$HOME/arm-linux-androideabi-4.9-vanilla/android-21 \
+    --prefix=/$HOME/xbmc-depends
```

Comment:
This is quite helpful too. Currently I refer to `tools/depends/Readme.md`, which is drastically out of date. Did you notice this too? Updating that Readme would also be helpful.